universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -153,6 +156,25 @@ public FieldWithLabel setAttribute(String attribute) {
     return this;
   }
 
+  public FieldWithLabel setMin(OptionalLong value) {
+    if (!this.fieldType.equals("number")) {
+      throw new RuntimeException(
+          "setting an OptionalLong min value is only available on fields of type `number`");
```

Comment:
Change the tics around number to single quotes, here and below.

Reply:
done